package/MDAnalysis/coordinates/TRZ.py

```diff
@@ -591,7 +571,11 @@ def _write_next_frame(self, obj):
                           "".format(", ".join(faked_attrs)))
 
         # Convert other stuff into our format
-        unitcell = triclinic_vectors(ts.dimensions).reshape(9)
+        if ts.dimensions is not None:
+            unitcell = triclinic_vectors(ts.dimensions).reshape(9)
+        else:
+            unitcell = np.zeros(9, dtype=np.float32)
```

Comment:
This behaviour should be at least documented, maybe warned too (I can't find any information about the TRZ file format..). I'll raise an issue...

Reply:
See #3328